fetch/main.js

```diff
@@ -0,0 +1,26 @@
+async function getData() {
+  try {
+    const response = await fetch('https://jsonplaceholder.typicode.com/users');
+    const data = await response.json();
```

Comment:
You have to check response.ok before getting the JSON since if it's not OK there probably won't be any JSON.